test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/NuGetExperimentationServiceTests.cs

```diff
@@ -77,17 +77,19 @@ public void IsEnabled_WithEnabledFlight_WithExperimentalService_ReturnsTrue()
             service.IsExperimentEnabled(ExperimentationConstants.PackageManagerBackgroundColor).Should().BeTrue();
         }
 
-        [Fact]
-        public void IsEnabled_WithEnvVarWithIncorrectValue_WithExperimentalService_ReturnsFalse()
+        [Theory]
+        [InlineData(true, true)]
+        [InlineData(false, false)]
+        public void IsEnabled_WithEnvVarNotSet_WithExperimentalService_ReturnsFlightEnabledValue(bool isFlightEnabled, bool expectedResult)
```

Comment:
These names are getting a bit overloaded. It'd be great if these could be renamed to be more concise and match the coding guideline of just 3 sections.
https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-method-naming